### PR TITLE
Recipe for chimichurri sauce.

### PR DESCRIPTION
**Pull request by mookerji**
/cc zmagg

### CODE REVIEW

File: sauces/2017-04-29-chimichurri-sauce.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#### Chimichurri Sauce
+
+This sauce really brings the Beef out of beef, particularly a skillet ribeye.
+
+Add to a mason jar, food processed until fine:
+
+- 1 cup packed fresh parsley, washed and dried
```
Comment (zmagg) [inline sauces/2017-04-29-chimichurri-sauce.md:7]: 2 cups.

File: sauces/2017-04-29-chimichurri-sauce.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- 5 medium cloves garlic, peeled
+- 2 tablespoons oregano leaves (if available)
+- ¼ cup sherry vinegar
```
Comment (zmagg) [inline sauces/2017-04-29-chimichurri-sauce.md:10]: less sherry vinegar, more like 2 tablespoons. Optionally steeped with chilis.

File: sauces/2017-04-29-chimichurri-sauce.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#### Chimichurri Sauce
+
+This sauce really brings the Beef out of beef, particularly a skillet ribeye.
+
+Add to a mason jar, food processed until fine:
+
+- 1 cup packed fresh parsley, washed and dried
+- 5 medium cloves garlic, peeled
```
Comment (zmagg) [inline sauces/2017-04-29-chimichurri-sauce.md:8]: One head garlic

File: sauces/2017-04-29-chimichurri-sauce.md (hunk 1 above)
Comment (mookerji) [reply sauces/2017-04-29-chimichurri-sauce.md:7]: Got it

File: sauces/2017-04-29-chimichurri-sauce.md (hunk 3 above)
Comment (mookerji) [reply sauces/2017-04-29-chimichurri-sauce.md:8]: oh man

File: sauces/2017-04-29-chimichurri-sauce.md (hunk 2 above)
Comment (mookerji) [reply sauces/2017-04-29-chimichurri-sauce.md:10]: Of course

**Comment by mookerji** [comment]
@zmagg <3

**Pull request merged**